packages/inferencer/src/inferencers/antd/list.tsx

```diff
@@ -152,21 +152,29 @@ export const renderer = ({
                 imports.push(["TagField", "@refinedev/antd"]);
                 let val = "item";
 
-                if (field?.relationInfer) {
-                    const valSingle = `${variableName}?.find((resourceItems) => resourceItems.id === ${accessor(
-                        "item",
-                        undefined,
-                        field.accessor,
-                    )})`;
-                    const valViewableSingle = accessor(
-                        valSingle,
-                        undefined,
-                        field?.relationInfer?.accessor,
-                    );
-                    val = valViewableSingle;
-                }
+                const cannotRender =
+                    field?.relationInfer &&
+                    field?.relationInfer?.type === "object" &&
+                    !field?.relationInfer?.accessor;
 
-                render = `render={(value: any[]) => ${loadingCondition} (
+                if (cannotRender) {
+                    render = `render={(value) => <span title="Inferencer failed to render this field (Cannot find key)">Cannot Render</span>}`;
```

Comment:
```suggestion
                    render = `render={(value) => <span title="Inferencer failed to render this field: ${JSON.stringify(value)} (Cannot find key)">Cannot Render</span>}`;
```

Maybe we can mention value in the message for convenience.